datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -1164,11 +1164,10 @@ public void handleMessage(final @NonNull Message msg) {
                         final Condition condition = lock.newCondition();
                         final StopSynchronizer synchronizationReceiver = new StopSynchronizer(lock, condition);
                         try {
-                            // To make sure the background service is stopped, we unbind this service
-                            // from it via the stopService method (to reduce code duplicity). As the
-                            // background service stopped itself in advance, we expect no active service:
-                            Validate.isTrue(!dataCapturingService.stopService(new Measurement(measurementId),
-                                    synchronizationReceiver));
```

Comment:
I had to remove the Validation as the background service was still running (still bound) so the app would crash.

Reply:
Shouldn't the validation work in that case? (stopService returns true since the service was still running and was successfully stopped?)

Reply:
Well, it should return true because it's already in the stop-universe and unbound by the stopService command. But I didn't dare to validate for this ;-) I'll add the validation again and check if it works locally and add a test to ensure we don't break it later on. (As we currently don't test the spaceWarning strategy implemented by our app.) To be continued....

Reply:
manuelly tested: works